### PR TITLE
Fix quiz import and also small change to question import

### LINKED ISSUE #243: Lista quizów - weryfikacja importu
Sprawdzenie czy quizy które się importują są prawidłowe i odrzucanie błędów

### LINKED ISSUE #312: import pytania/quizu/statystyk z pustymi polami
można importować pytania z pustymi polami, np. tytuł

### PR DESCRIPTION
**Pull request by Karolk99**

### CODE REVIEW

File: src/lecturer/components/quizzesListView/QuizzesListView.tsx [hunk 1]
```diff
@@ -1,25 +1,140 @@
...
+
     const getContainer: () => FrontQuiz[] = () => {
         return store.quizzes;
     }
     const setContainer = (value: TitleType[]) => {
         store.quizzes = value as FrontQuiz[];
     }
 
+    const onImport = (e: ProgressEvent<FileReader>) => {
+        console.log("start");
```
Comment (michal367) [inline src/lecturer/components/quizzesListView/QuizzesListView.tsx]: remove console.log

File: src/lecturer/components/quizzesListView/QuizzesListView.tsx [hunk 2]
```diff
@@ -1,25 +1,140 @@
...
+                }
+
+                if (correctQuiz){
+                    resultQuiz.push(quiz);
+                }
+            }
+            store.quizzes = [...store.quizzes, ...resultQuiz];
+            setMessage("Zaimportowano " + resultQuiz.length + " z " + counter + " quizów");
+            setSnackbarState({ ...snackbarState, open: true });
+            console.log("end");
```
Comment (michal367) [inline src/lecturer/components/quizzesListView/QuizzesListView.tsx]: remove console.log

**Review by michal367 — CHANGES_REQUESTED** [review]
separate checking functions (`isQuestionCorrect`, `isQuizCorrect` - they should check if given question/quiz is correct and return true/false) from those importing functions and place them in some file in `lecturer/util` folder

I can import quiz without questions (empty questions array) and I should not

**Review by Petros9 — APPROVED** [review]
good

**Review by michal367 — APPROVED** [review]

**Pull request merged**